labs/iot/gps-lab/client/code/raspberry-pi-gps-tracker.py

```diff
@@ -0,0 +1,95 @@
+import time
+import serial
+
+# Using the Python Device SDK for IoT Hub:
+#   https://github.com/Azure/azure-iot-sdk-python
+# The sample connects to a device-specific MQTT endpoint on your IoT Hub.
+from azure.iot.device import IoTHubDeviceClient, Message
+
+# The device connection string to authenticate the device with your IoT hub.
+CONNECTION_STRING = "<Device_Connection_String>"
+
+# Define the JSON message to send to IoT Hub.
+MSG_TXT = '{{"latitude": {latitude},"longitude": {longitude}}}'
+
+SERIAL_PORT = "/dev/serial0"
+
+gps = serial.Serial(SERIAL_PORT, baudrate = 9600, timeout = 1.5)
+
+#This method converts a transmitted string from DDMM.MMMMM to DD.MMMM
+def formatDegreesMinutes(coordinates, digits):
```

Comment:
Are there any Python libraries to help with formatting the messages from a GPS device?

Reply:
Agree. Changed the code to use https://github.com/Knio/pynmea2. It make the parsing much simpler.